README.md

```diff
@@ -55,6 +55,7 @@ Supported keys of the map are:
 ## License
 
 Copyright © 2016 Coda Hale
+Copyright © 2017 Sentry
```

Comment:
Perfect, what's your preferred process for that? Should I just push a clone to `getsentry` and you redirect people there or is this a first class thing on Github?

And thanks a lot for the library btw. I'm no Clojure expert but it's the best one out there and Clojure is just-different-enough (vs say Kotlin/Scala) that "just use the Java SDK" is crappy advice, as you know.